sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -138,8 +143,25 @@ def as_dict(self):
 class CounterMetric(Metric):
   def __init__(self, counter_dict, submit_timestamp, metric_id):
     super(CounterMetric, self).__init__(submit_timestamp, metric_id)
-    self.value = counter_dict.committed
     self.label = str(counter_dict.key.metric.name)
+    self.value = counter_dict.committed
+
+
+class DistributionMetrics(Metric):
+  def __init__(self, dists, submit_timestamp, metric_id, key):
+    super(DistributionMetrics, self).__init__(submit_timestamp, metric_id)
+    self.dist_label = str(key)
```

Comment:
Can we expect callers to pass an `str` to this constructor to avoid input type ambiguity and str conversion here?

Reply:
Sure, makes sense.